src/metrics/commits-per-pr/index.ts

```diff
@@ -10,9 +8,11 @@ import { Commits, CommitsPerPrOutput, CommitsPerPrPayload } from "./interfaces";
 import { getTimestamp } from "../../shared/get-timestamp";
 import { LogErrors } from "../../shared/log-messages";
 import { logger } from "../../core";
+import { Octokit } from "@octokit/rest";
 
 export const collectCommitsPerPrMetrics = async (
-  triggerEvent: SignedTriggerEvent
+  triggerEvent: SignedTriggerEvent,
+  repoClient: Octokit
```

Comment:
to avoid unused var in some metrics

we would need to make this optional too
```suggestion
  repoClient?: Octokit
```